security-updates/osv/index.rst

```diff
@@ -1159,18 +1159,18 @@ The ``ranges`` field specifies the ``type`` of versioning scheme being used in
 an ecosystem and the events of when a vulnerability was ``introduced``,
 ``fixed``, and any type-specific fields.
 
-Since Debian versioning is not particularly a type supported in OSV yet, the
-``type`` will always be ``ECOSYSTEM``.
+Since Debian versioning is not yet a supported type in OSV, the ``type`` will
+always be ``ECOSYSTEM``.
 
-For Ubuntu, the Ubuntu Security Team does not currently track when a
-vulnerability was first introduced. Instead, we focus on identifying out of
-the shipped source package versions, which are vulnerable to such CVE.
-Therefore, the ``introduced`` field is always ``0``. And whenever the team
-patches a vulnerability, the source package version is listed in ``fixed``.
+The Ubuntu Security Team doesn't currently track when a vulnerability was
+first introduced. Instead, we focus on identifying which of the shipped source
```

Comment:
This may be inaccurate, I believe we have started including this data in our CVE files but I need to double check.